src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotifyClaimCallbackHandler.java

```diff
@@ -293,4 +297,37 @@ private boolean areAnyRespondentsLitigantInPerson(CaseData caseData) {
         return caseData.getRespondent1Represented() == NO
             || (YES.equals(caseData.getAddRespondent2()) ? (caseData.getRespondent2Represented() == NO) : false);
     }
+
+    private LocalDateTime getServiceDate(CaseData caseData) {
+        if (Objects.nonNull(caseData.getCosNotifyClaimDefendant1())
+            && Objects.nonNull(caseData.getCosNotifyClaimDefendant2())) {
+
+            if (caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant()
+                .isEqual(caseData.getCosNotifyClaimDefendant2().getCosDateOfServiceForDefendant())) {
+
+                return  caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant().atStartOfDay();
+
+            } else if (caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant()
+                .isBefore(caseData.getCosNotifyClaimDefendant2().getCosDateOfServiceForDefendant())) {
+
+                return  caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant().atStartOfDay();
+
+            } else if (caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant()
+                .isAfter(caseData.getCosNotifyClaimDefendant2().getCosDateOfServiceForDefendant())) {
+
+                return caseData.getCosNotifyClaimDefendant2().getCosDateOfServiceForDefendant().atStartOfDay();
+            }
+
+        } else if (Objects.isNull(caseData.getCosNotifyClaimDefendant1())
+            && Objects.nonNull(caseData.getCosNotifyClaimDefendant2())) {
+
+            return  caseData.getCosNotifyClaimDefendant2().getCosDateOfServiceForDefendant().atStartOfDay();
+
+        } else if (Objects.nonNull(caseData.getCosNotifyClaimDefendant1())
+            && Objects.isNull(caseData.getCosNotifyClaimDefendant2())) {
+
+            return caseData.getCosNotifyClaimDefendant1().getCosDateOfServiceForDefendant().atStartOfDay();
+        }
+        return time.now();
+    }
 }
```

Comment:
only need compare those 3 dates, time.now(), cos notify claim date 1 and cos notify claim date 1.  if date does not exist then ignore, return the earliest